<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodecParams</class>
 <widget class="QDialog" name="CodecParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Codec Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/icons.qrc">
    <normaloff>:/icons/MainIconTx.svg</normaloff>:/icons/MainIconTx.svg</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="GroupBoxInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Codec Information</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="Logo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="res/icons.qrc">:/icons/LogoOpus.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="LabelInfoDescription">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Description</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="LabelVersion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Version: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LabelInfoVersion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>_ver_</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="LabelInfoWebSite">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Website</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="GroupBoxApplication">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Application</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="RadioButtonAppVOIP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>VoIP</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupApplication</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonAppAudio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Audio</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupApplication</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupBoxSignal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Signal</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="RadioButtonSignalVoice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Voice</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupSignal</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonSignalMusic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Music</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupSignal</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupBoxFEC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Forward Error Correction</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="RadioButtonFECOff">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Off</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupFEC</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonFECOn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>On</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupFEC</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="GroupBoxChannels">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Channels</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="RadioButtonChannelsMono">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Mono</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupChannels</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonChannelsStereo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Stereo</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupChannels</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupBoxBandwidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Bandwidth</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="RadioButtonBandwidthNB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Narrow Band (4 kHz)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupBandwidth</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonBandwidthMB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Medium Band (6 kHz)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupBandwidth</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonBandwidthWB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Wide Band (8 kHz)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupBandwidth</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonBandwidthSWB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Super Wide Band (12 kHz)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupBandwidth</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButtonBandwidthFB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Full Band (20 kHz)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroupBandwidth</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>RadioButtonChannelsMono</tabstop>
  <tabstop>RadioButtonChannelsStereo</tabstop>
  <tabstop>RadioButtonBandwidthNB</tabstop>
  <tabstop>RadioButtonBandwidthMB</tabstop>
  <tabstop>RadioButtonBandwidthWB</tabstop>
  <tabstop>RadioButtonBandwidthSWB</tabstop>
  <tabstop>RadioButtonBandwidthFB</tabstop>
  <tabstop>RadioButtonAppVOIP</tabstop>
  <tabstop>RadioButtonAppAudio</tabstop>
  <tabstop>RadioButtonSignalVoice</tabstop>
  <tabstop>RadioButtonSignalMusic</tabstop>
  <tabstop>RadioButtonFECOff</tabstop>
  <tabstop>RadioButtonFECOn</tabstop>
 </tabstops>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="ButtonGroupChannels"/>
  <buttongroup name="ButtonGroupSignal"/>
  <buttongroup name="ButtonGroupFEC"/>
  <buttongroup name="ButtonGroupBandwidth"/>
  <buttongroup name="ButtonGroupApplication"/>
 </buttongroups>
</ui>
